### PR TITLE
Link: Apple to Delay new Features and Focus on Software Quality

### PR DESCRIPTION
**Pull request by tjdraper**

### CODE REVIEW

File: _posts/2018-01-31-apple-focusing-on-quality.markdown [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+title: Apple to Delay new Features and Focus on Software Quality
+author: tjdraper
+link_url: https://www.bloomberg.com/news/articles/2018-01-30/apple-is-said-to-push-back-some-key-iphone-software-features
+tags: [ Apple, iOS, macOS ]
+date: 2018-01-31 9:00
+---
+Mark Gurman writing for Bloomberg:
```
Comment (smithtimmytim) [inline _posts/2018-01-31-apple-focusing-on-quality.markdown]: The publication should be italicized.

File: _posts/2018-01-31-apple-focusing-on-quality.markdown [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+>
+> As part of an annual release of new iPhone models, Apple also usually rolls out a major iOS update each year. The current software version, iOS 11, added augmented-reality features, a file management app and business-user enhancements for the iPad. For iOS 12, Apple has been working on additions like a redesigned home screen app grid, a multiplayer mode for augmented reality games, and a merger of the third-party applications running on iPhones and Macs, the people said, asking not to be named discussing information that isn’t public.
+
+There has been a lot of talk for the last few years as each release cycle ramps up about another "Snow Leopard Moment." And in fact, while Apple didn't spell it out in those exact words, the name High Sierra, and the sentiment behind it was certainly meant to be a "Snow Leopard" like release. That hasn't really panned out as High Sierra is easily the buggiest and most frustrating release of macOS in many years. So if this rumor turns out to be true, I will be both happy, but also somewhat embarrassed for Apple. High Sierra was supposed to be that release.
```
Comment (smithtimmytim) [inline _posts/2018-01-31-apple-focusing-on-quality.markdown]: I think the sentence works better like this: 

> If this rumor turns out to be true, I will be both happy, and somewhat embarrassed for Apple

File: _posts/2018-01-31-apple-focusing-on-quality.markdown [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+And iOS has simply never had a moment to catch its breath. It's never had its "Snow Leopard" moment. If you ask me, even with the much greater resources Apple devotes to iOS, that lack of time to catch their breath is starting to show. You can't push software forward when all you ever do is roll out new features and get them "good enough" to ship and then immediately start on the next new batch of features. iOS is starting to feel less refined than it used to. Honestly, as much I far prefer the direction the design took when they departed from the heavy skeuomorphism, it doesn't feel like this new look and feel has ever quite settled and received the polish and finalization it needs. And the whole OS feels less stable than it used to. When iOS 11 was first released my 6s battery life was a disaster. This has happened before on some level—but what happened to my battery with iOS 11 as different. I couldn't make it much past noon. And it wasn't until one release ago that it finally settled down. And here we go with another round of that in a few months? No thank you!
```
Comment (smithtimmytim) [inline _posts/2018-01-31-apple-focusing-on-quality.markdown]: I wouldn't start the paragraph with "and"

Comment (smithtimmytim) [inline _posts/2018-01-31-apple-focusing-on-quality.markdown]: I think the sentence works better like this:

> You can't push software forward when all you ever do is roll out new features, get them "good enough" to ship, and then immediately start on the next new batch of features.

Comment (smithtimmytim) [inline _posts/2018-01-31-apple-focusing-on-quality.markdown]: > what happened to my battery with iOS 11 *is* different

You start a few sentences with "and" here. Some are back to back.

Also, this should probably be two paragraphs.

**Comment by tjdraper** [comment]
@smithtimmytim Round 2.

**Pull request merged**